docs/_includes/components/wells.html

```diff
@@ -2,32 +2,32 @@
   <h1 id="wells" class="page-header">Wells</h1>
 
   <h2>Default well</h2>
-  <p>Use the well as a simple effect on an element to give it an inset effect.</p>
+  <p>Use the well as a simple effect on an element to make it a little informative card.</p>
   <div class="bs-example" data-example-id="default-well">
-    <div class="well">
+    <div class="well-xs">
```

Comment:
Don't we want to encourage `well well-xs` instead?

Reply:
No, that's styling, this is responsive. Thing about col behaviour, or text-center behaviour